doc/Support/Support-New-OS.md

```diff
@@ -1,2 +1,2 @@
 source: Support/Support-New-OS.md
-<meta http-equiv="refresh" content="0; url=/Developing/Support-New-OS/" />
+<meta http-equiv="refresh" content="0; url=/Developing/Support-New-OS/__" />
```

Comment:
What's the reason for the __?

Reply:
probably attempting to type in another window and I missed it.  Thanks.